lib/presentation/widgets/details_widget.dart

```diff
@@ -0,0 +1,140 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:marvel/data/models/character.dart';
+import 'package:marvel/data/models/series.dart';
+import 'package:marvel/presentation/features/details/bloc/details_bloc.dart';
+import 'package:marvel/presentation/features/details/bloc/details_state.dart';
+
+class DetailsWidget extends StatelessWidget {
+  final int characterId;
+
+  const DetailsWidget(this.characterId, {Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocBuilder<DetailsBloc, DetailsState>(
+      builder: (context, state) {
+        final loading = state.loading;
+        final character = state.character;
+        final allSeries = state.allSeries;
+        final error = state.error;
+        if (loading == true) {
+          return const Center(
+            child: CircularProgressIndicator(),
+          );
+        }
+        if (loading == false && character != null && allSeries != null) {
+          return Scaffold(
+            appBar: AppBar(
+              centerTitle: true,
+              title: Text(
+                character.name,
+                style: const TextStyle(
+                  color: Colors.white,
+                  fontSize: 24,
+                ),
+              ),
+            ),
+            body: SingleChildScrollView(
+              child: Column(
+                mainAxisAlignment: MainAxisAlignment.start,
+                children: [
+                  _buildImage(character.thumbnail),
+                  if (character.description.isNotEmpty) const Text('DESCRIPTION', style: TextStyle(fontSize: 20)),
```

Comment:
Description title and description itself is one block, so we can use it like 
```
if (character.description.isNotEmpty) _buildDescription(character.description);
```
and then in `_buildDescription` create a Column with the title and content.

For now it looks like you will still show the empty description without title if it is empty

Reply:
done